flow/raster_cache.cc

```diff
@@ -115,7 +117,11 @@ sk_sp<SkImage> RasterCache::GetPrerolledImage(GrContext* context,
         (is_complex || isWorthRasterizing(picture))) {
       TRACE_EVENT2("flutter", "Rasterize picture layer", "width",
                    physical_size.width(), "height", physical_size.height());
-      SkImageInfo info = SkImageInfo::MakeN32Premul(physical_size);
+      SkImageInfo info = SkImageInfo::Make(physical_size.width(),
+                                           physical_size.height(),
+                                           kRGBA_8888_SkColorType,
+                                           kPremul_SkAlphaType,
+                                           sk_ref_sp(dst_color_space));
```

Comment:
Yes sk_sp is the preference.  We want to hide constructors and call sk_sp factories.  However, sk_sp is new in Skia so not everything has been updated yet.

Similarly, when we pass ref counted objects as parameters, if we are going to ref them, they should be passed as sk_sp.  If they are not going to be reffed, we can/should use a bare ptr.

This can be a little confusing because the "old" way of doing things was to always pass a bare ptr and then comment about reffing.
`// This function will ref parameter x`

I suspect you ran into one of the "old" cases - which is probably a good reminder for us to update the API.

----------------

Anyway, my code here is not good...  The getter gives back a bare ptr so as not to force us to ref it.  But since we will, I should go ahead and ref it and pass around sk_sp.  And use std::move() to minimize ref count churn.